userguide/sharing.rst

```diff
@@ -468,48 +468,98 @@ application.
    <https://tinyurl.com/archive-zfs-over-nfs-vmware>`__.
 #endif freenas
 
-Before creating an NFS share, go to
-:menuselection:`Services --> NFS` and click the sliding button to
-turn on the service.
-
-Next, go to :menuselection:`Storage --> Pools` to create a dataset
-for the share. Refer to :ref:`Adding Datasets` for more information
-about dataset creation.
-
 Create an NFS share by going to
 :menuselection:`Sharing --> Unix (NFS) Shares`
 and clicking |ui-add|. :numref:`Figure %s <nfs_share_wiz_fig>` shows
 an example of creating an NFS share.
 
-For the required :guilabel:`Path` field, browse to the dataset created
-for the share. In the :guilabel:`Comment` field enter a share name, note
-that if left empty, the share name is the list of selected Path entries.
-Click the :guilabel:`SAVE` button to create the share.
-
 .. _nfs_share_wiz_fig:
 
 .. figure:: images/sharing-unix-nfs-add.png
 
    NFS Share Creation
 
 
-Edit an NFS share by going to
-:menuselection:`Sharing --> Unix (NFS)`, clicking the |ui-options|
-button for the desired share, then :guilabel:`Edit`. In the example
-shown in :numref:`Figure %s <nfs_share_settings_fig>`, the configuration
-screen is open for the *nfs_share1* share.
+Remember these points when creating NFS shares:
 
-.. _nfs_share_settings_fig:
+#. Clients will specify the :guilabel:`Path` when mounting the share.
 
-.. figure:: images/sharing-unix-nfs-edit-example.png
+#.  The :guilabel:`Maproot` and :guilabel:`Mapall` options are
+    exclusive, meaning only one can be used as the |web-ui| does
+    not allow both. The :guilabel:`Mapall` options supersede the
+    :guilabel:`Maproot` options. To restrict only the *root* user
+    permissions, set the :guilabel:`Maproot` option. To restrict
+    permissions of all users, set the :guilabel:`Mapall` options.
 
-   NFS Share Settings
+#.  Each pool or dataset is considered to be its own filesystem and
+    individual NFS shares cannot cross filesystem boundaries. Sharing
+    multiple directories with an additional paths is only allowed within
```

Comment:
tense: paths-->path

or should it be "Sharing multiple directories with additional paths ... "?